ci/azure-test.yml

```diff
@@ -1,6 +1,9 @@
 jobs:
 # Check formatting
 - job: ${{ parameters.name }}
+  variables:
+    cfgRelease: ''
+    cfgReleaseChannel: ${{ parameters.rust }}
```

Comment:
I don't think RLS will build on stable/beta in the nearest future so I think we should just globally set these variables for simplicity and call it a day